cwt/draft-ietf-ace-cbor-web-token.xml

```diff
@@ -1144,12 +1175,13 @@ d08343a1010aa1054d99a0d7846e762c49ffe8a63e0b5858b918a11fd81e438b
 	  <figure align="center" anchor="fig:exampleNestedHex"
 		title='Signed and Encrypted CWT as hex string'>
 	    <artwork><![CDATA[
-d08343a1010aa1054d86bbd41cc32604396324b7f38058a372439fbff538aa7b
-601ebfb29454050a3c99fd13b27216d084556496c7355c4bb462510f8e0e8479
-dbe08722d620e96bcb7764d75140d96220f062679b46b897e7abe0c325dc2c96
-d8bb2c8334e3b92a42c0078983e753c054e647ad5387ed149f802f52b5a95ebf
-5f153c4fd64854ab7531e082b7f22721f939d257c94f8bc248e1d9cf04f9dd4e
-5de7ab62df37842fabec230a657d4abf7162bc786345ebb8eb3af0
+d08343a1010aa2044c53796d6d6574726963313238054d4a0694c0e69ee6b595
+6655c7b258b7f6b0914f993de822cc47e5e57a188d7960b528a747446fe12f0e
+7de05650dec74724366763f167a29c002dfd15b34d8993391cf49bc91127f545
+dba8703d66f5b7f1ae91237503d371e6333df9708d78c4fb8a8386c8ff09dc49
+af768b23179deab78d96490a66d5724fb33900c60799d9872fac6da3bdb89043
+d67c2a05414ce331b5b8f1ed8ff7138f45905db2c4d5bc8045ab372bff142631
+610a7e0f677b7e9b0bc73adefdcee16d9d5d284c616abeab5d8c291ce0
```

Comment:
Why did the length of this change?

Reply:
because the signed object that is encrypted got a kid attribute too